ToDoLite/DetailViewController.m

```diff
@@ -226,6 +246,18 @@ - (UITableViewCell *)couchTableSource:(CBLUITableSource *)source cellForRowAtInd
                                                     forIndexPath:indexPath];
     CBLQueryRow *row = [source rowAtIndex:indexPath.row];
     Task *task = [Task modelForDocument:row.document];
+
+    int64_t delay = 1.5; // In seconds
```

Comment:
Ahh .. makes sense.  Can you put a brief comment above the usages of dispatch_after to that effect?